components/raftstore-v2/src/raft/peer.rs

```diff
@@ -510,10 +510,6 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
                 "apply_index" =>  self.storage().entry_storage().applied_index()
             );
             self.set_replay_watch(None);
-            // Flush to avoid recover again and again.
```

Comment:
This part is removed:
1. The direct reason is that this makes `test_data_recovery()` test fail. Since, `test_data_recovery()` makes some assertions on number of sst files after node restart.
2. But, after deliberated considerations, this actually should be removed. This optimization is no longer optimized because this may generates many small files after reducing the `PAUSE_FOR_REPLAY_GAP`.